zuul.d/content_provider.yaml

```diff
@@ -16,18 +16,35 @@
       - ^kuttl-test.yaml$
     required-projects:
       - opendev.org/zuul/zuul-jobs
+      - openstack-k8s-operators/ansible-microshift-role
+      - openstack-k8s-operators/ci-framework
+      - openstack-k8s-operators/cinder-operator
+      - openstack-k8s-operators/data-plane-adoption
+      - openstack-k8s-operators/dataplane-operator
+      - openstack-k8s-operators/designate-operator
+      - openstack-k8s-operators/edpm-ansible
+      - openstack-k8s-operators/edpm-image-builder
+      - openstack-k8s-operators/glance-operator
+      - openstack-k8s-operators/heat-operator
+      - openstack-k8s-operators/horizon-operator
+      - openstack-k8s-operators/infra-operator
       - openstack-k8s-operators/install_yamls
-      - openstack-k8s-operators/openstack-operator
-      - github.com/openstack-k8s-operators/ci-framework
```

Comment:
https://review.rdoproject.org/codesearch/?q=github.com%2Fopenstack-k8s-operators%2Fci-framework&i=nope&literal=nope&files=&excludeFiles=&repos=

https://review.rdoproject.org/codesearch/?q=github.com%2Fopenstack-k8s-operators%2Frepo-setup&i=nope&literal=nope&files=&excludeFiles=&repos=

we refer to these repos with the githab.com in the name string. idk if this will have any impact?

Reply:
I guess it's working for the dataplane operator?

Reply:
I had the same concerns, but @viroel mentioned that at the end all projects are installed into `src/github.com/<name_of_project>` directories.

Reply:
I spent more time trying to understand what's the difference between `github.com/openstack-k8s-operators/ci-framework` and `openstack-k8s-operators/ci-framework`.
I found we're using both naming schemes, like here: https://review.rdoproject.org/r/plugins/gitiles/config/+/refs/heads/master/zuul.d/openstack-ci-base.yaml#133

I'm still not sure where it's being used and how ZUUL interprets them.

Reply:
If we use `openstack-k8s-operators/ci-framework` or `github.com/openstack-k8s-operators/ci-framework`, both will work. But if `openstack-k8s-operators/ci-framework` is referenced in multiple Zuul connection. Then it will cause issue while clonning the correct project . I would suggest we can use the full name.

Reply:
Removed `openstack-k8s-operators/ci-framework` and `openstack-k8s-operators/repo-setup` and restored `github.com` versions of both.